src/components/features/charactersAndCampaigns/MovesSection/MovesSection.tsx

```diff
@@ -56,6 +56,7 @@ export function MovesSection(props: MovesSectionProps) {
         sx={{
           overflow: "auto",
           flexGrow: 1,
+          '&::-webkit-scrollbar': { width: 0 }
```

Comment:
Does this hide the scrollbar? I would revert these if so - in some accessibility cases, browser scrollbars can be the only method for some users to scroll.

Reply:
reverted